mobilecoind-json/src/bin/main.rs

```diff
@@ -306,7 +306,7 @@ fn read_address_code(
     format = "json",
     data = "<transfer>"
 )]
-fn transfer(
+fn build_and_submit(
```

Comment:
If we're changing this the documentation needs to be updated too

Reply:
This just changes the function name. The URL remains the same. What documentation are you referring to that is sensitive to the function name being changed?

Reply:
Oh, never mind, you're right